data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
@Zbizu I mean, according to link provide in #2017, in tibia.wikia.com:

> Since update 10.92 you can no longer "use" a furniture kit to open it, and it will not cause a spark effect anymore if you try to. In order to open it, you now need to right-click the furniture package and choose "unwrap". An item can be rewrapped by right-clicking it and choosing "wrap".
so actions script should be removed and handled in Wrap event.